### ISSUE TITLE
Now that Universal addons are a thing ...

### ISSUE THREAD

**Issue by Ludovicus-Maior** [BigWigsMods/packager#99]
Could the packager be taught to use the .TOC files for each client and produce a zip file with only the files for that release?
We currently have an addon with a significant amount of data per client that is different, with a core addon which is truly universal.

I suspect others might be in the same situation.

--Ludovicus

**Comment by Ludovicus-Maior** [comment]
If this more is active, perhaps the other .toc files should go away and the appropriate client specific .toc file be renamed to the primary .toc name, to it becomes clean this is no longer a universal addon.

**Comment by Nevcairiel** [comment]
You can do that today already with pkgmeta directives.

The goal for the community should be one zip that works for all clients going forward, no longer separate zips, which could already be done before.

**Comment by Ludovicus-Maior** [comment]
@Nevcairiel Currently our BCC release has 500 files in it, 250 of which are not loaded for classic or retail.  The amount not loaded would increase with each client supported, since Blizzard has tweaked quests, npc locations and the like for every release.
If Universal Zips for all clients are the **right way forward** despite the deadweight for some addons, then so be it.

**Comment by Tuller** [comment]
I think using differing pkgmetas would be the best approach in your case.

**Comment by Ludovicus-Maior** [comment]
@Gethe game me an example from https://github.com/Haleth/Aurora .
TL;DR:  use different .pkgmeta files for each release and have the release specific stuff in different directories, then use ignore: - Foo/Retail in the yaml file to skip the irrelevant stuff. 

This suffices.  If we want to go this way, this a reasonable way to do it.

Thanks @Nevcairiel and @Tuller .
